app/stac_api/management/commands/remove_expired_items.py

```diff
@@ -0,0 +1,68 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.management.base import CommandParser
+from django.utils import timezone
+
+from stac_api.models import Asset
+from stac_api.models import Item
+from stac_api.utils import CommandHandler
+from stac_api.utils import CustomBaseCommand
+
+
+class Handler(CommandHandler):
+
+    def delete(self, instance, object_type):
+        if self.options['dry_run']:
+            self.print_success(f'skipping deletion of {object_type} {instance}')
+        else:
+            instance.delete()
+
+    def run(self):
+        # print(self.options)
+        self.print_success('running command to remove expired items')
+        min_age_hours = self.options['min_age_hours']
+        self.print_warning(f"deleting all items expired longer than {min_age_hours} hours")
+        items = Item.objects.filter(
+            properties_expires__lte=timezone.now() - timedelta(hours=min_age_hours)
+        ).all()
+        for i in items:
```

Comment:
Just personal taste: I find `i` as an iterator suboptimal (except maybe for simple numeric for loops). But I'm very curious about other opinions!

Reply:
I usually choose names based on the scope the variable is used.
Usually very few letters if it is only used in the next few lines, more letters/descriptive names if it used used "further away". In this case I see that "i" is also often used as index, so I am also fine with 4 letters "item".